package.json

```diff
@@ -95,7 +95,7 @@
     "codecov": "CODECOV=1 ./scripts/test/js_test.sh",
     "watch": "WATCH=1 ./scripts/test/js_test.sh",
     "flow": "flow check",
-    "fmt": "prettier-eslint --write --no-semi --ignore 'static/js/flow/**/*.js' 'static/js/**/*.js'",
+    "fmt": "LOG_LEVEL= prettier-eslint --write --no-semi --ignore 'static/js/flow/**/*.js' 'static/js/**/*.js'",
```

Comment:
When I run `docker-compose run watch fmt` I get an error since `LOG_LEVEL` is set to `INFO` in `.env` and `.env.example`. I figured it didn't matter here so I just cleared it for this command